ts-tests/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "ts-tests",
   "version": "0.0.0",
   "scripts": {
+    "build": "truffle compile",
     "test": "jest"
```

Comment:
Realised that we can consider having `truffle compile && jest` since the tests rely on the contract compilation.

Reply:
i think good to combine them in one script